scripts/stablex/setup_environment_for_single_trade.js

```diff
@@ -0,0 +1,37 @@
+// Prerequisit for this script:
+// Have an account funded with Rinkeby OWL 
+// This simple script will set up a trade between OWL and A New Token
+const StablecoinConverter = artifacts.require("StablecoinConverter")
+const ERC20Mintable = artifacts.require("ERC20Mintable.sol")
+const { addTokens, depositTokens, placeOrder } = require("./utilities.js")
+
+module.exports = async function (callback) {
+  try {
+
+    const instance = await StablecoinConverter.deployed()
+    const accounts = await web3.eth.getAccounts()
+    const account = accounts[0]
+
+    const TokenOWL = artifacts.require("TokenOWL")
+    const owlToken = await TokenOWL.at(await instance.feeToken.call())
+
+    const amount = (10 ** 18).toString()
```

Comment:
I find it slightly strange that this works. If 10**18 returns a number that can be cast to a string, why would we need to cast it? Also, maybe `bn.js` would be better here.